docs/architecture/notes.md

```diff
@@ -28,114 +28,114 @@ These components are:
 
 ### Metadata
 
-> Additional note information.
+> Additional `Note` information.
 
-Notes include metadata such as the sender’s account ID and a [tag](#note-discovery) that aids in discovery. Regardless of [storage mode](#note-storage-mode), these metadata fields remain public.
+`Note`s include metadata such as the sender’s account ID and a [tag](#note-discovery) that aids in discovery. Regardless of [storage mode](#note-storage-mode), these metadata fields remain public.
 
 ## Note Lifecycle
 
 ![Architecture core concepts](../img/architecture/note/note-life-cycle.png)
 
-The note lifecycle proceeds through four primary phases: **creation**, **validation**, **discovery**, and **consumption**. Throughout this process, notes function as secure, privacy-preserving vehicles for asset transfers and logic execution.
+The `Note` lifecycle proceeds through four primary phases: **creation**, **validation**, **discovery**, and **consumption**. Throughout this process, `Note`s function as secure, privacy-preserving vehicles for asset transfers and logic execution.
 
 ### Note creation
 
-Accounts can create notes in a transaction. The note exists if it is included in the global Notes DB.
+Accounts can create `Note`s in a transaction. The `Note` exists if it is included in the global `Note`s DB.
 
 - **Users:** Executing local or network transactions.
-- **Miden operators:** Facilitating on-chain actions, e.g. such as executing user notes against a DEX or other contracts.
+- **Miden operators:** Facilitating on-chain actions, e.g. such as executing user `Note`s against a DEX or other contracts.
 
 #### Note storage mode
 
-As with [accounts](accounts.md), notes can be stored either publicly or privately:
+As with [accounts](accounts.md), `Note`s can be stored either publicly or privately:
 
-- **Public mode:** The note data is stored in the [note database](https://0xpolygonmiden.github.io/miden-base/architecture/state.html#notes-database), making it fully visible on-chain.
-- **Private mode:** Only the note’s hash is stored. The note’s actual data remains off-chain, enhancing privacy.
+- **Public mode:** The `Note` data is stored in the [note database](https://0xpolygonmiden.github.io/miden-base/architecture/state.html#notes-database), making it fully visible on-chain.
+- **Private mode:** Only the `Note`’s hash is stored. The `Note`’s actual data remains off-chain, enhancing privacy.
 
 #### Ephemeral note
 
-These use-case specific notes can be consumed even if not yet validated by being chained together into one final proof. This can allow for example sub second communication below blocktimes by adding additional trust assumptions.
+These use-case specific `Note`s can be consumed even if not yet validated by being chained together into one final proof. This can allow for example sub second communication below blocktimes by adding additional trust assumptions.
```

Comment:
```suggestion
These `Note`s can be consumed even if not yet registered on-chain. They can be chained together into one final proof. This can allow, for example, sub, sub-second communication below blocktimes by adding additional trust assumptions.
```